gensrc/proto/internal_service.proto

```diff
@@ -494,6 +494,17 @@ message PPulsarProxyResult {
     optional PPulsarBacklogBatchProxyResult pulsar_backlog_batch_result = 102;
 };
 
+message PGetFileSchemaRequest {
+    // unused, just for preventing jprotobuf error "no field use annotation @com.baidu.bjf.remoting.protobuf.annotation.Protobuf ..."
+    optional int32 pad = 1;
```

Comment:
should contain credential information, and can refer to `FileTableScanNode#cloudConfiguration`

Reply:
The credential information is attached as attachment.